templates/entrydetails.mustache

```diff
@@ -0,0 +1,72 @@
+{{!
+    This file is part of Moodle - https://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template mod_cpdlogbook/entrydetails
+
+    Template purpose and description.
+
+    Classes required for JS:
+    * none
+
+    Data attributes required for JS:
+    * none
+
+    Context variables required for this template:
+    * none
+
+    Example context (json):
+    {
+        "name": "Reading",
+        "completiondate": "Fri, 3 Dec 2021",
+        "points": 3,
+        "duration": "1 hour",
+        "location": "Brisbane",
+        "summary": "I read a lot of things",
```

Comment:
Can you format_text this the same as how it is done in the table so it honor line breaks and simple links, eg use this as a test summary:

```
I presented on a range of topics around gold fish, and other aquatic animal species.

https://example.com
```